lib/context.js

```diff
@@ -37,6 +37,14 @@ module.exports = async function getContext() {
     case 'GitHub Actions':
       return context;
     case 'Bitbucket Pipelines': {
+      const baseContext = {
+        ref: process.env.BITBUCKET_TAG
+          ? `refs/tags/${process.env.BITBUCKET_TAG}`
+          : `refs/heads/${process.env.BITBUCKET_BRANCH}`,
+        sha: process.env.BITBUCKET_COMMIT,
```

Comment:
is this the correct `sha` to use?

this is the head commit:
`$BITBUCKET_COMMIT`

this is the last commit ([ref](https://community.atlassian.com/t5/Bitbucket-questions/Getting-the-destination-commit-SHA-for-a-PR-in-Pipelines/qaq-p/1450021#M57162)):
`git rev-parse origin/$BITBUCKET_PR_DESTINATION_BRANCH`

Reply:
Isn't the head commit and the last commit the same thing?

Reply:
This hasn't changed from the previous code currently in production: https://github.com/readmeio/readme-micro/blob/109acc3503c7dbe27c7c04c4594dd62cc5c2e9f2/lib/context.js#L59